float32 v_x
float32 v_y
float32 v_z
float32 v_yaw
---
bool result